### ISSUE TITLE
How preproccess the image when use the pretrained model?

### ISSUE THREAD

**Issue by lidehuihxjz** [moskomule/senet.pytorch#47]
Could you please tell me how to preproccess the image when use the pretrained SE-ResNet50 model? I use the preproccessing as showed below, but get wrong classify output.

_Compose([
    Resize((244,244)),
    ToTensor(),
    Normalize([.485, .456, .406], [.229, .224, .225]),
])_

**Comment by moskomule** [comment]
I think just following the evaluation phase is fine.
